### DISCUSSION TITLE
Why are new items not copied to "current tab"?

### DISCUSSION THREAD

**Discussion by jwezel** [hluk/CopyQ#2682] [Ideas]
There is the "current tab" (the one selected in the tab page).

But then there is a config item `clipboard_tab` that specifies where new clipboard content will go.

This makes it rather awkward to switch tabs for storing new clipboard items.

To me it would make much more sense to put new clipboard items in the "current tab", not in a tab that has to be set in the preferences.

**Comment by hluk** [comment]
I prefer not to have other tabs cluttered with semi-random clipboard content.

You can change the behavior by adding the following command ([here is how to add the command to CopyQ](http://copyq.readthedocs.io/en/latest/faq.html#how-to-load-shared-commands-and-share-them)):

```ini
[Command]
Command="
    const option = 'clipboard_tab';

    var onTabSelected_ = global.onTabSelected;
    global.onTabSelected = function() {
        settings(option, selectedTab());
        return onTabSelected_()
    }

    var onClipboardChanged_ = global.onClipboardChanged;
    global.onClipboardChanged = function() {
        const clipboardTab = settings(option);
        if (clipboardTab)
            setData(mimeOutputTab, clipboardTab);
        return onClipboardChanged_();
    }"
Icon=
IsScript=true
Name=Store Clipboard to the Current Tab
```

**Reply by jwezel** [reply]
Thanks. But now no content is getting stored anymore.

**Reply by hluk** [reply]
Do you use the latest version of the app? Anything in the logs (F12 shortcut from the main window)?
